cmd/main.go

```diff
@@ -337,7 +337,15 @@ func setupReconcilers(mgr ctrl.Manager, setupLog logr.Logger,
 	}
 
 	if kserveState == "managed" {
-		if err := setupInferenceGraphReconciler(mgr); err != nil {
+		authConfigCrdAvailable, authCrdErr := utils.IsCrdAvailable(
```

Comment:
nit: since `authConfigCrdAvailable` does not seems to be used elsewhere, wonder if this check should be moved to `SetupWithManager` function where it is actually in use

Reply:
that can be done
i first added it in `InferenceGraphReconciler.SetupWithManager()`, but then seems they wanna have the check move as higher as possible, so it was eventually done in the main